content/sql/create-table-foreign-key.md

```diff
@@ -9,12 +9,14 @@ github: https://github.com/zernonia
 
 Add foreign key to product_id using `references`.
 
-`on delete cascade` When a referenced foreign key is deleted or updated, all rows referencing that key are deleted or updated, respectively.
+`on delete cascade` When a referenced foreign key is deleted, all rows referencing that key are deleted, respectively.
+
+`on update cascade` When a referenced foreign key is updated, all rows referencing that key are updated, respectively.
 
 ```sql
 create table table_name (
   id bigint generated by default as identity primary key,
-  product_id uuid references products (id) on delete cascade,
+  product_id uuid references products (id) on delete cascade on update cascade,
```

Comment:
I think it will be better if `on update cascade` is placed on another sql script below, instead of appending it to the end.

Reply:
I usually use `on delete cascade on update cascade` to make it more optimal and avoid errors